manager-parent/manager-v1/src/test/resources/application.properties

```diff
@@ -25,10 +25,20 @@ event-bridge.kafka.client.secret=client-secret
 event-bridge.kafka.security.protocol=PLAINTEXT
 event-bridge.kafka.sasl.mechanism=PLAIN
 
+# RHOAS Client
+event-bridge.rhoas.instance-api.host=http://test-instance-api.redhat.com
+event-bridge.rhoas.mgmt-api.host=http://test-management-api.redhat.com
+# RHOAS Client is unused by -core so Quarkus removes it from Arc unless we flag it as unremovable.
+quarkus.arc.unremovable-types=com.redhat.service.smartevents.rhoas.RhoasClient
```

Comment:
I understand the comment but not the reason: if this is not used in `-core` why do we need to force to preserve it?

Reply:
Well... I can only assume Quarkus was pruning it as there is no direct usage. Without this setting for tests I get a warning that the class has been pruned and subsequently an unresolved type error. I can investigate more when I'm back at my laptop tomorrow.